pyanaconda/ui/gui/hubs/__init__.py

```diff
@@ -352,7 +361,14 @@ def refresh(self):
         GUIObject.refresh(self)
         self._createBox()
 
-        GLib.timeout_add(100, self._update_spokes)
+        for hub in Hub._hubs_collection:
+            if hub.timeout_id is not None:
+                log.debug("Disable event loop for hub %s", hub.__class__.__name__)
```

Comment:
Just a nitpick, but I would change this to "Disabling event loop for hub %s". :)